pkg/e2e/operator/pod_placement_config_test.go

```diff
@@ -295,4 +296,83 @@ var _ = Describe("The Multiarch Tuning Operator", Serial, func() {
 			Entry(utils.MasterNodeSelectorLabel, utils.MasterNodeSelectorLabel),
 		)
 	})
+	Context("When a v1beta1 pod placement config is created", func() {
+		It("should create v1beta1 CPPC and get v1alpha1 with plugin specs", func() {
+			By("Creating a v1beta1 ClusterPodPlacementConfig")
+			err := client.Create(ctx,
+				NewClusterPodPlacementConfig().
+					WithName(common.SingletonResourceObjectName).
+					WithNodeAffinityScoring(true).
+					WithNodeAffinityScoringTerm(utils.ArchitectureArm64, 50).
+					Build(),
+			)
+			Expect(err).NotTo(HaveOccurred(), "failed to create ClusterPodPlacementConfig v1beta1", err)
+			// Get the details
+			ppc := &v1beta1.ClusterPodPlacementConfig{}
+			err = client.Get(ctx, runtimeclient.ObjectKeyFromObject(&v1beta1.ClusterPodPlacementConfig{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: common.SingletonResourceObjectName,
+				},
+			}), ppc)
+			Expect(err).NotTo(HaveOccurred(), "failed to get ClusterPodPlacementConfig v1beta1", err)
```

Comment:
I see lines 325 and 326, if you want to keep that verification do it right after getting the v1beta1 object back from the API:

```suggestion
			By("Get the v1beta1 version of the CPPC")
			ppc := &v1beta1.ClusterPodPlacementConfig{}
			err = client.Get(ctx, runtimeclient.ObjectKeyFromObject(&v1beta1.ClusterPodPlacementConfig{
				ObjectMeta: metav1.ObjectMeta{
					Name: common.SingletonResourceObjectName,
				},
			}), ppc)
			Expect(err).NotTo(HaveOccurred(), "failed to get ClusterPodPlacementConfig v1beta1", err)
			By("Validate the plugins stanza is set")
			Expect(ppc.Spec.Plugins).NotTo(BeNil())
			Expect(ppc.Spec.Plugins.NodeAffinityScoring.IsEnabled().To(BeTrue())
			
```